test/networkd-test.py

```diff
@@ -918,8 +918,7 @@ def test_search_domains(self):
                 break
             time.sleep(0.1)
         self.assertRegex(contents, 'search .*one two three four')
-        self.assertNotIn('seven\n', contents)
-        self.assertIn('# Too many search domains configured, remaining ones ignored.\n', contents)
+        self.assertNotIn('# Too many search domains configured, remaining ones ignored.\n', contents)
```

Comment:
This tests seems completely superfluous. I would simply drop it. Are you sure you didn't want to invert the condition in `self.assertNotIn('seven\n', contents)` instead?

Reply:
This is the only test that checks that "search domains are written out into resolv.conf". So I'd like to keep it.

Will adjust to ensure that it tests that _all_ search domains are written out.